<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportWindow</class>
 <widget class="QMainWindow" name="ReportWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>738</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_12">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>2</number>
        </property>
        <widget class="QWidget" name="tab_russian">
         <attribute name="title">
          <string>Русские буквы</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QTableWidget" name="tableWidget_russian">
            <attribute name="horizontalHeaderVisible">
             <bool>false</bool>
            </attribute>
            <attribute name="horizontalHeaderDefaultSectionSize">
             <number>57</number>
            </attribute>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>false</bool>
            </attribute>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
            <attribute name="verticalHeaderDefaultSectionSize">
             <number>80</number>
            </attribute>
            <attribute name="verticalHeaderStretchLastSection">
             <bool>false</bool>
            </attribute>
            <row>
             <property name="text">
              <string>Новая строка</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>Новая строка</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>Новая строка</string>
             </property>
            </row>
            <column>
             <property name="text">
              <string>Новый столбец</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Новый столбец</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Новый столбец</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Новый столбец</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Новый столбец</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Новый столбец</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Новый столбец</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Новый столбец</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Новый столбец</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Новый столбец</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Новый столбец</string>
             </property>
            </column>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_2">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout">
                  <item>
                   <widget class="QLabel" name="label_words_rus">
                    <property name="font">
                     <font>
                      <bold>true</bold>
                     </font>
                    </property>
                    <property name="text">
                     <string>Количество букв:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="lineEdit_words">
                    <property name="font">
                     <font>
                      <bold>true</bold>
                     </font>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_3">
                  <item>
                   <widget class="QLabel" name="label_paragraph_rus">
                    <property name="font">
                     <font>
                      <bold>true</bold>
                     </font>
                    </property>
                    <property name="text">
                     <string>Количество абзацев:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="lineEdit_paragraph">
                    <property name="font">
                     <font>
                      <bold>true</bold>
                     </font>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_2">
                  <item>
                   <widget class="QLabel" name="label_proposal_rus">
                    <property name="font">
                     <font>
                      <bold>true</bold>
                     </font>
                    </property>
                    <property name="text">
                     <string>Количество предложений:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="lineEdit_proposal">
                    <property name="font">
                     <font>
                      <bold>true</bold>
                     </font>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Orientation::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>98</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QLabel" name="label_glasn_rus">
                  <property name="font">
                   <font>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>Количество гласных:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_glasn">
                  <property name="font">
                   <font>
                    <bold>true</bold>
                   </font>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item>
                 <widget class="QLabel" name="label_soglasn_rus">
                  <property name="font">
                   <font>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>Количество согласных:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_soglasn">
                  <property name="font">
                   <font>
                    <bold>true</bold>
                   </font>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <item>
                 <widget class="QLabel" name="label_zaglav_rus">
                  <property name="font">
                   <font>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>Количество заглавных:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_zaglav">
                  <property name="font">
                   <font>
                    <bold>true</bold>
                   </font>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_english">
         <attribute name="title">
          <string>Латинские буквы</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_10">
          <item>
           <widget class="QTableWidget" name="tableWidget_english">
            <attribute name="horizontalHeaderVisible">
             <bool>false</bool>
            </attribute>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
            <row>
             <property name="text">
              <string>Новая строка</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>Новая строка</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>Новая строка</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>Новая строка</string>
             </property>
            </row>
            <column>
             <property name="text">
              <string>Новый столбец</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Новый столбец</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Новый столбец</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Новый столбец</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Новый столбец</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Новый столбец</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Новый столбец</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Новый столбец</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Новый столбец</string>
             </property>
            </column>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_17">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_18">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_8">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_19">
                  <item>
                   <widget class="QLabel" name="label_words_eng">
                    <property name="font">
                     <font>
                      <bold>true</bold>
                     </font>
                    </property>
                    <property name="text">
                     <string>Количество букв:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="lineEdit_words_3">
                    <property name="font">
                     <font>
                      <bold>true</bold>
                     </font>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_20">
                  <item>
                   <widget class="QLabel" name="label_paragraph_eng">
                    <property name="font">
                     <font>
                      <bold>true</bold>
                     </font>
                    </property>
                    <property name="text">
                     <string>Количество абзацев:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="lineEdit_paragraph_3">
                    <property name="font">
                     <font>
                      <bold>true</bold>
                     </font>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_21">
                  <item>
                   <widget class="QLabel" name="label_proposal_eng">
                    <property name="font">
                     <font>
                      <bold>true</bold>
                     </font>
                    </property>
                    <property name="text">
                     <string>Количество предложений:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="lineEdit_proposal_3">
                    <property name="font">
                     <font>
                      <bold>true</bold>
                     </font>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
              <item>
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Orientation::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>98</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_9">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_22">
                <item>
                 <widget class="QLabel" name="label_glasn_eng">
                  <property name="font">
                   <font>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>Количество гласных:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_glasn_3">
                  <property name="font">
                   <font>
                    <bold>true</bold>
                   </font>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_23">
                <item>
                 <widget class="QLabel" name="label_soglasn_eng">
                  <property name="font">
                   <font>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>Количество согласных:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_soglasn_3">
                  <property name="font">
                   <font>
                    <bold>true</bold>
                   </font>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_24">
                <item>
                 <widget class="QLabel" name="label_zaglav_eng">
                  <property name="font">
                   <font>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>Количество заглавных:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_zaglav_3">
                  <property name="font">
                   <font>
                    <bold>true</bold>
                   </font>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_symbol">
         <attribute name="title">
          <string>Символы</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_11">
          <item>
           <widget class="QTableWidget" name="tableWidget_symbol">
            <attribute name="horizontalHeaderVisible">
             <bool>false</bool>
            </attribute>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
            <row>
             <property name="text">
              <string>Новая строка</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>Новая строка</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>Новая строка</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>Новая строка</string>
             </property>
            </row>
            <column>
             <property name="text">
              <string>Новый столбец</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Новый столбец</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Новый столбец</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Новый столбец</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Новый столбец</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Новый столбец</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Новый столбец</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Новый столбец</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_back">
        <property name="text">
         <string>Назад</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
